src/adaptations/device-layer/ESP32/ESP32Config.cpp

```diff
@@ -40,27 +41,32 @@ const char ESP32Config::kConfigNamespace_WeaveConfig[]                     = "we
 const char ESP32Config::kConfigNamespace_WeaveCounters[]                   = "weave-counters";
 
 // Keys stored in the weave-factory namespace
-const ESP32Config::Key ESP32Config::kConfigKey_SerialNum               = { kConfigNamespace_WeaveFactory, "serial-num"         };
-const ESP32Config::Key ESP32Config::kConfigKey_DeviceId                = { kConfigNamespace_WeaveFactory, "device-id"          };
-const ESP32Config::Key ESP32Config::kConfigKey_DeviceCert              = { kConfigNamespace_WeaveFactory, "device-cert"        };
-const ESP32Config::Key ESP32Config::kConfigKey_DevicePrivateKey        = { kConfigNamespace_WeaveFactory, "device-key"         };
-const ESP32Config::Key ESP32Config::kConfigKey_ProductRevision         = { kConfigNamespace_WeaveFactory, "product-rev"        };
-const ESP32Config::Key ESP32Config::kConfigKey_ManufacturingDate       = { kConfigNamespace_WeaveFactory, "mfg-date"           };
-const ESP32Config::Key ESP32Config::kConfigKey_PairingCode             = { kConfigNamespace_WeaveFactory, "pairing-code"       };
+const ESP32Config::Key ESP32Config::kConfigKey_SerialNum                   = { kConfigNamespace_WeaveFactory, "serial-num"         };
+const ESP32Config::Key ESP32Config::kConfigKey_ManufAttestDeviceId         = { kConfigNamespace_WeaveFactory, "ma-device-id"       };
```

Comment:
I would prefer it if we left the string name the same here.  Otherwise people are going to need to wipe their devices.

Given that the values are in a "WeaveFactory" namespace, I think the unadorned name "device-id" won't be too ambiguous.